packages/office-ui-fabric-react/src/components/Announced/examples/Announced.QuickActions.Example.tsx

```diff
@@ -53,177 +52,162 @@ export interface IAnnouncedQuickActionsExampleItem {
+              aria-label="Show actions"
+              styles={iconButtonStyles}
+              menuProps={{
+                items: [
+                  {
+                    key: 'delete',
+                    text: 'Delete',
+                    onClick: () => deleteItem(index),
+                  },
+                  {
+                    key: 'rename',
+                    text: 'Rename',
+                    onClick: () => renameItem(item, index),
+                  },
+                ],
+              }}
+            />
+          </div>
+        );
+      } else {
+        return <span>{fieldContent}</span>;
+      }
+    },
+    [deleteItem, renameItem],
+  );
+
+  const updateItemName = useConstCallback((index: number) => {
+    if (textField && textField.current) {
+      const renamedItems = items;
+      renamedItems[index].name = textField.current.value || renamedItems[index].name;
+      setItems([...renamedItems]);
+      setRenameDialogOpen(false);
+      setAnnounced(<Announced message="Item renamed" aria-live="assertive" />);
+      setPreviousAnnouncedValue(announced);
+    }
+  });
 
-    this._async = new Async(this);
+  const closeRenameDialog = useConstCallback((): void => {
+    setRenameDialogOpen(false);
+  });
 
-    // Populate with items for demos.
-    if (_items.length === 0) {
+  // Populate with items for demos.
+  React.useEffect(() => {
+    if (items.length === 0) {
       for (let i = 0; i < 20; i++) {
-        _items.push({
+        items.push({
           key: i,
           name: 'Item ' + i,
           modified: getMockDateString(),
-          modifiedby: _names[Math.floor(Math.random() * _names.length)],
+          modifiedby: names[Math.floor(Math.random() * names.length)],
           filesize: Math.floor(Math.random() * 30).toString() + ' MB',
         });
       }
     }
-
-    this._selection = new Selection({});
-
-    this.state = {
-      items: _items,
-      renameDialogOpen: false,
-      dialogContent: undefined,
-      announced: undefined,
-    };
-  }
-
-  public componentDidUpdate(prevState: IAnnouncedQuickActionsExampleState) {
-    if (prevState.announced !== this.state.announced && this.state.announced !== undefined) {
-      this._async.setTimeout(() => {
-        this.setState({
-          announced: undefined,
-        });
+  }, [items]);
+
+  // componentDidUpdate
+  React.useEffect(() => {
+    if (previousAnnouncedValue !== announced && announced !== undefined) {
+      setTimeout(() => {
+        setAnnounced(undefined);
+        setPreviousAnnouncedValue(announced);
       }, 2000);
     }
-  }
-
-  public componentWillUnmount(): void {
-    this._async.dispose();
-  }
+  }, [announced, previousAnnouncedValue, setTimeout]);
 
-  public render(): JSX.Element {
-    const { items, renameDialogOpen, dialogContent } = this.state;
-
-    return (
-      <>
-        {this._renderAnnounced()}
+  return (
+    <>
+      {announced}
+      <MarqueeSelection selection={selection}>
```

Comment:
Get rid of the MarqueeSelection here since none of the actions actually work on multi-select (I did this recently in the class version of the example)